wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Message/TwitterEmbed.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Wrapper around Twitter's createTweet API.
+ *
+ * @author	Tim Duesterhus
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module	WoltLabSuite/Core/Ui/Message/TwitterEmbed
+ */
+define(['https://platform.twitter.com/widgets.js'], function(Widgets) {
+	"use strict";
+	
+	var twitterReady = new Promise(function(resolve, reject) {
+		twttr.ready(resolve);
+	});
+	
+	/**
+	 * @exports	WoltLabSuite/Core/Ui/Message/TwitterEmbed
+	 */
+	return {
+		/**
+		 * Embed the tweet identified by the given tweetId into the given container.
+		 * 
+		 * @param {HTMLElement} container
+		 * @param {string} tweetId
+		 * @param {boolean} clearContents Whether to remove existing children of the given container after embedding the tweet.
```

Comment:
```suggestion
		 * @param {boolean} removeChildren Whether to remove existing children of the given container after embedding the tweet.
```
Somehow the refactoring didn't rename the variable in the comment.